x-pack/filebeat/input/awss3/sqs_test.go

```diff
@@ -24,6 +24,7 @@ import (
 const testTimeout = 10 * time.Second
 
 var errFakeConnectivityFailure = errors.New("fake connectivity failure")
+var errFakeGetAttributeFailute = errors.New("something went wrong")
```

Comment:
`s/errFakeGetAttributeFailute/errFakeGetAttributeFailure/g` and aggregate into a single decl.